docs/content/docs/deep/container-and-component-loading.md

```diff
@@ -25,225 +25,295 @@ as well as the finished diagram in [Appendix 1](#appendix-1).
-`ComponentContext` **(7)** and executes the corresponding `instantiateComponent` with the `ComponentContext` **(8.1)**.
-
-![Image 7](/images/container-and-component-loading-7.jpg)
-
-You might notice a lot of similarities between the `ContainerRuntime` creation flow and the `ComponentRuntime`
+The important thing to note here is that up until this point the Hosting Application and Fluid know nothing of the Fluid
+`ContainerRuntime` or the `DataStore` code.
+That code is provided after the `Container` is established and stored in the op stream.
+**This is powerful because it allows the Hosting Applications to load Containers without knowing the underlying code.**
+This is how Teams and Outlook can easily load the Fluid preview app `Container` and `DataStore`s.
+
+![Image 5](images/container-and-component-loading-5.jpg)
+
+The implementer of `instantiateRuntime()` is what we refer to as a "Container Developer".
+As you can see, the term is slightly overloaded since they are not actually writing the `Container` object, but the
+`ContainerRuntime`, along with a factory to create it.
+That factory should be an implementation of `IContainerFactory` and thus have the `instantiateRuntime()` function.
+The `Container` specifically looks for an exported `fluidExport` property within the code loaded by the `CodeLoader`,
+which it expects to implement `IContainerFactory`.
+
+The `instantiateRuntime()` function can do any number of things but has become primarily responsible for **(6.2)**:
+
+1. Creating the `ContainerRuntime` object.
+2. Setting the `request` handler on the `ContainerRuntime`.
+    - The `request` handlers are used to route requests through the `Container` (more on this later).
+    - The primary use is to get DataStores.
+3. Providing a `DataStoreRegistry` of DataStore Factories to the `ContainerRuntime`.
+    - The `DataStoreRegistry` is basically a map from names(types) of `DataStore`s to the factories that can create them.
+      It defines which `DataStore`s can be created in the `Container`
+4. Creating the default `DataStore`.
+
+![Image 6](images/container-and-component-loading-6.jpg)
+
+Containers can exist without `DataStore`s but they are not very functional.
+The paradigm we've created is for the Container Developer (`instantiateRuntime()` implementer) to create a default
+`DataStore`.
+The default `DataStore` is simply the first `DataStore` in the `Container`.
+It is created the same as every other `DataStore`, with the only difference being that it is created in the
+`instantiateRuntime()` call as opposed to being created by another `DataStore` (more on this later).
+
+Having a default `DataStore` allows the Hosting Application to make a `request` against the `Container` asking for the
+default `DataStore` without knowing what the default `DataStore` is (also more on this later).
+
+**NOTE**: the `request` handler is still supported but is in the process of being superseded by a different mechanism,
```

Comment:
Should this (and the below) be put inside a `NOTE` or `WARNING` callout block for emphasis?